reactor-stream/src/test/java/reactor/rx/StreamTests.java

```diff
@@ -640,6 +645,16 @@ private void parallelMapManyTest(String dispatcher, int iterations) throws Inter
 		System.out.println("");
 	}
 
+	/**
+	 * See https://github.com/reactor/reactor/issues/451
+	 */
+	@Test
+	public void partitionByHashCodeShouldNeverCreateMoreStreamsThanSpecified() throws Exception {
+		Stream<Integer> stream = Streams.range(-10, 10).map(Long::intValue);
```

Comment:
use integer because an integer hash code is itself, hence ensuring some negative hashcodes